### PR TITLE
added wrappers and API callers for /application endpoint

### PR DESCRIPTION
**Pull request by BhalchandraNaik**
dependent Galileo PR : https://bitbucket.org/fitpay/galileo-flutter-app/pull-requests/68/custom-building-the-kyc-form

### CODE REVIEW

File: lib/src/api.dart [hunk 1]
```diff
@@ -908,6 +908,18 @@ class API {
     throw response.statusCode;
   }
 
+  Future<KycApplication> getKycApplication(Uri uri) async {
+    var response = await _httpClient.get("https://${uri.toString()}", headers: await _headers());
```
Comment (ssteveli) [inline lib/src/api.dart]: @bpwalford and/or @leif-waldner -  This URLs coming out of edge-3 need to be fixed, this is bad being here ^^

Comment (bpwalford) [reply lib/src/api.dart]: @leif-waldner is on it

File: lib/src/models.dart [hunk 2]
```diff
@@ -909,22 +909,22 @@ class KycApplication extends BaseResource {
...
-  factory KycApplication.fromJson(Map<String, dynamic> json) => _$KycApplicationFromJson(json);
+  factory Application.fromJson(Map<String, dynamic> json) => _$ApplicationFromJson(json);
 
-  Map<String, dynamic> toJson() => _$KycApplicationToJson(this);
+  Map<String, dynamic> toJson() => _$ApplicationToJson(this);
 }
 
 @JsonSerializable(nullable: true)
-class KycApplicationSteps extends BaseResource {
+class KycSteps extends BaseResource {
```
Comment (ssteveli) [inline lib/src/models.dart]: I think `ApplicationSteps` would be a more appropriate name.

File: lib/src/api.dart (hunk 1 above)
Comment (BhalchandraNaik) [reply lib/src/api.dart]: I have removed the patch to include `https://` in the  `uri`. Now using `uri` directly

File: lib/src/models.dart (hunk 2 above)
Comment (BhalchandraNaik) [reply lib/src/models.dart]: done!

**Pull request merged**